corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -305,13 +313,13 @@ function AddressEntry(question, options) {
             });
         }
 
+        var defaultGeocoderLocation = initialPageData('default_geocoder_location') || {};
+
         var geocoder = new MapboxGeocoder({
-            accessToken: window.MAPBOX_ACCESS_TOKEN,
-            types: 'address',
+            accessToken: MAPBOX_ACCESS_TOKEN,
+            types: "address",
             enableEventLogging: false,
-            // proximity set to NYC
-            proximity: { longitude: -74.006058, latitude: 40.712772},
-            getItemValue: self.geocoderItemCallback,
+            proximity: defaultGeocoderLocation.coordinates,
```

Comment:
You can also do a check if `defaultGeocoderLocation.coordinates` and call `setProximity` https://github.com/mapbox/mapbox-gl-geocoder/blob/master/API.md#setproximity. I'm assuming an undefined proximity does not have negative sideeffects? What does it default to?

Reply:
docs don't say much about undefined proximity, so I did what you suggested.